bc_obps/service/facility_service.py

```diff
@@ -16,9 +16,120 @@
 from registration.models import User
 from django.db import transaction
 from django.utils import timezone
+from registration.models import Address
 
 
 class FacilityService:
```

Comment:
I don't see pytests for the new things that were added to this service.

Reply:
Apologies, I am unsure of the meaning of "I don't see pytests for the new things..." Could you expand on your thought?